Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
 import './App.css';
-
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+import HeaderContainer from "./components/HeaderComponents/HeaderContainer"
+import CardContainer from "./components/CardComponents/CardContainer"
+  const App = () => {
+    return <div>
+      <HeaderContainer />;
+      <CardContainer />;
```

Comment:
You don't need semi-colons after you render a component, these are showing up in your page.